dallinger/frontend/static/scripts/tracking/load-tracker.js

```diff
@@ -0,0 +1,65 @@
+/*global require */
+/*jshint esversion: 6 */
+
+var dlgr = window.dlgr = (window.dlgr || {});
+
+if (window.getUrlParameter === undefined) {
+  var getUrlParameter = function getUrlParameter(sParam) {
+      var sPageURL = decodeURIComponent(window.location.search.substring(1)),
+          sURLVariables = sPageURL.split("&"),
+          sParameterName,
+          i;
+
+      for (i = 0; i < sURLVariables.length; i++) {
+          sParameterName = sURLVariables[i].split("=");
+
+          if (sParameterName[0] === sParam) {
+              return sParameterName[1] === undefined ? true : sParameterName[1];
+          }
+      }
+  };
+}
+
+(function (getUrlParameter, require) {
+
+  var Scribe = require('./scribe-analytics.min');
+  var ScribeDallinger = require('./scribe-dallinger');
+
+  function getParticipantId() {
```

Comment:
This may also be available via the new Dallinger JS module.

Reply:
I don't think there's a method for this, in the module, but even if there is we need to define it separately here for the same reason as above (ideally we'd have a shared library module that we could compile into both modules via webpack).

Reply:
Okay.